StreetDrop/StreetDrop/Presentation/MusicDropView/View/MusicDropViewController.swift

```diff
@@ -0,0 +1,274 @@
+//
+//  MusicDropViewController.swift
+//  StreetDrop
+//
+//  Created by 맹선아 on 2023/05/18.
+//
+
+import UIKit
+
+import SnapKit
+import RxSwift
+
+final class MusicDropViewController: UIViewController {
+    private var viewModel: MusicDropViewModel
+    private let disposeBag: DisposeBag = DisposeBag()
+
+    init(viewModel: MusicDropViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .black
+
+        configureHierarchy()
+        configureLayout()
+        viewModel.fetchAlbumCover()
+        viewModel.fetchAdress()
+        bindViewModel()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        makeViewIntoGradientCircle()
+    }
+
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        self.commentTextView.endEditing(true)
+    }
+
+    //MARK: - 디자인 요소 (레이아웃 잡힌 후 그라데이션 layer 적용)
+    private var topGradientCircleView: UIView = UIView()
+    private var smallerCenterGradientCircleView: UIView = UIView()
+    private var LargerCenterGradientCircleView: UIView = UIView()
+
+    //MARK: - 뷰 아이템 요소
+    private let locationLabel: UILabel = UILabel(
+        textAlignment: .center,
+        font: .title2,
+        numberOfLines: 2
+        // 👉 남은이슈: 주소만 글씨색 다르게하기
+    )
+
+    private let albumImageView: UIImageView = UIImageView(
+        cornerRadius: 10
+    )
+
+    private let musicNameLabel: UILabel = UILabel(
+        font: .body
+    )
+
+    private let artistLabel: UILabel = UILabel(
+        font: .caption1
+    )
+
+    private let musicInfoStackView: UIStackView = UIStackView(
+        alignment: .center,
+        spacing: 5
+    )
+
+    private let commentTextView: UITextView = UITextView(
+        backgroundColor: .darkGray,
+        cornerRadius: 10,
+        inset: 10
+    )
+
+    private let CommentGuidanceLabel: UILabel = UILabel(
+        font: .caption2
+    )
+
+    private let commentStackView: UIStackView = UIStackView(
+        spacing: 5
+    )
+
+    private lazy var dropButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitleColor(.white, for: .normal)
+        button.layer.cornerRadius = 10
+        button.backgroundColor = .darkGray
+        button.addAction(touchedUpDropButton(), for: .touchUpInside)
```

Comment:
addAction대신 RxCocoa를 통해 구현하면 좋을 것 같습니다 ! 추후에 수정 부탁드립니다!
ex)
```suggestion
        button.rx.tap
        .bind {
            touchedUpDropButton() 메소드 기능
        }
```